flink-staging/flink-streaming/flink-streaming-core/src/main/java/org/apache/flink/streaming/api/graph/StreamingJobGraphGenerator.java

```diff
@@ -93,6 +96,15 @@ public JobGraph createJobGraph(String jobName) {
 		
 		configureCheckpointing();
 
+		try {
+			InstantiationUtil.writeObjectToConfig(
+			this.streamGraph.getExecutionConfig(),
+			this.jobGraph.getJobConfiguration(),
+			ExecutionConfig.CONFIG_KEY);
+			} catch (IOException e) {
+			throw new RuntimeException("Config object could not be written to Job Configuration: " + e);
```

Comment:
Can you replace the + by a , ?
Throwable.toString() does not really give useful output.